pallets/pallet-web3-names/src/lib.rs

```diff
@@ -496,6 +502,32 @@ pub mod pallet {
 		fn unban_name(name: &Web3NameOf<T>) {
 			Banned::<T>::remove(name);
 		}
+
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> DispatchResult {
+			// check if for each owner there is a name stored.
+			Owner::<T>::iter().try_for_each(
```

Comment:
Can we also test the reverse direction? For each entry here, there has to be a reverse entry in `Names`.